app/models/item.rb

```diff
@@ -6,4 +6,8 @@ class Item < ApplicationRecord
   has_many :orders, through: :order_items
 
   enum item_status: ['available', 'retired']
+
+  def get_quantity_in_order(order)
```

Comment:
I think if we wanted to clarify it even more we could call it get_item_quantity_in_order, but I think because its in the item model thats implicit. I would approve it as is for now.